broker-cli/commands/health-check.js

```diff
@@ -10,7 +10,8 @@ const { validations, handleError } = require('../utils')
  */
 const STATUS_CODES = Object.freeze({
   OK: 'OK',
-  UNKNOWN: 'UNKNOWN'
+  UNKNOWN: 'UNKNOWN',
+  VALIDATED: 'VALIDATED'
```

Comment:
seems weird to define this is a string when it should be available on the engine itself, right?

Reply:
@treygriffith this is definitely weird, but what this does is change the `VALIDATED` status to `OK`, since it was really weird to have a term that the user might not understand?

Reply:
Its essentially a UI change https://github.com/sparkswap/broker/pull/328/files#diff-54e861896da44d61a844920c19b0bc2eR46

Reply:
@treygriffith maybe the way to go is to change the `VALIDATED` status code to `OK` on the lnd-engine?